src/components/app/App.tsx

```diff
@@ -11,9 +11,13 @@ import HowItWorks from "./pages/HowItWorks";
 import Ambassadors from "./pages/Ambassadors";
 import GetInvolved from "./pages/GetInvolved";
 
+import ReactPixel from "react-facebook-pixel";
 import { HashRouter as Router, Route, Switch } from "react-router-dom";
 import { trackPageOpen, load, page } from "../../utils/analytics";
 
+ReactPixel.init("601740554108493");
```

Comment:
we should not expose a key - can you add this to env file and change to process.env ?